apps/passport-server/migrations/33_login_rate_limit.sql

```diff
@@ -0,0 +1,3 @@
+alter table commitments 
+add column account_reset_timestamps timestamptz[] 
```

Comment:
I'm not an expert on SQL schemas, but I think the more flexible/normalized form of this would be to have this be a separate table with columns for user ID and timestamp, and let the "list" be represented by multiple rows.  I think that would make it easier to do things in parallel without conflicts (if/when you're worried about multiple servers).  You could also do things like delete stale timetamps in a single query without processing them in JS.

That would imply more complicated database queries/transactions, so may be overkill for this use case (where we may not care if the accounting is not quite right), but would be important if you needed to focus on transactional correctness.

Reply:
I agree that creating a table for this makes some sense, but I think i'm going to keep it this way until we actually do need to worry about transactional correctness if that's ok with you.